src/VMware/VMware/ChangeLog.md

```diff
@@ -19,6 +19,17 @@
 -->
 ## Upcoming Release
 
+## Version 0.8.0
+* Updated the AVS VMware cmdlets api version to `2024-09-01`. 
```

Comment:
Please move the release notes to under the Upcoming Release section and DO NOT create a new version number.